onnx/helper.py

```diff
@@ -1563,6 +1563,16 @@ def tensor_dtype_to_storage_tensor_dtype(tensor_dtype: int) -> int:
     """
     return mapping.TENSOR_TYPE_MAP[tensor_dtype].storage_dtype
 
+def tensor_dtype_to_storage_np_dtype(tensor_dtype: int) -> int:
```

Comment:
```suggestion
def tensor_dtype_to_storage_np_dtype(tensor_dtype: int) -> ...:
```

Looks like the type is not int?